lwt-unix/conduit_lwt_unix.ml

```diff
@@ -28,14 +28,23 @@ let () =
   with Not_found -> ()
 
 type tls_lib = | OpenSSL | Native | No_tls [@@deriving sexp]
-let tls_library = ref No_tls
-let () =
-    tls_library := try
-        match Sys.getenv "CONDUIT_TLS" with
-        | "native" | "Native" | "NATIVE" -> Native
-        | _ -> OpenSSL
-      with Not_found -> OpenSSL
-    (* TODO build time selection *)
+let default_tls_library =
+  try
+    match String.lowercase_ascii (Sys.getenv "CONDUIT_TLS") with
+    | "native" -> Native
+    | "openssl" | "libressl" -> OpenSSL
+    | "none" | "notls" -> No_tls
+    | _ -> OpenSSL
```

Comment:
Agreed!  I was trying to keep the changes relatively non-invasive.  I can make that syntactic change here if you'd like though.